lib/fpm/package/pacman.rb

```diff
@@ -0,0 +1,394 @@
+# -*- coding: utf-8 -*-
+require "fpm/package"
+require "backports"
+require "fileutils"
+require "find"
+
+class FPM::Package::Pacman < FPM::Package
+
+  opt_depends = []
+  option "--optdepends", "PACKAGE",
+    "Add an optional dependency to the pacman package." do |define|
+    opt_depends << define
+    next opt_depends
+  end
+
+  option "--use-file-permissions", :flag,
+    "Use existing file permissions when defining ownership and modes"
+
+  option "--user", "USER", "The owner of files in this package", :default => 'root'
+
+  option "--group", "GROUP", "The group owner of files in this package", :default => 'root'
+
+  # The list of supported compression types. Default is xz (LZMA2)
+  COMPRESSION_TYPES = [ "gz", "bzip2", "xz", "none" ]
+
+  option "--compression", "COMPRESSION", "The compression type to use, must " \
+    "be one of #{COMPRESSION_TYPES.join(", ")}.", :default => "xz" do |value|
+    if !COMPRESSION_TYPES.include?(value)
+      raise ArgumentError, "Pacman compression value of '#{value}' is invalid. " \
+        "Must be one of #{COMPRESSION_TYPES.join(", ")}"
+    end
+    value
+  end
+
+  def initialize(*args)
+    super(*args)
+    attributes[:pacman_opt_depends] = []
```

Comment:
Is this attribute for the `--optdepends` flag? If so, it should automatically show up as `:pacman_optdepends` as you have it defined. I don't think you'll need to do this, especially if you change the option to `:multivalued => true`

And also, since I think this is a typo, does this work? :P